src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.tsx

```diff
@@ -42,7 +52,7 @@ const VideoSpotlightBlock = ({
       <div className={s.bleedImageWrapper}>
         <div
           className={s.bleedImage}
-          style={{ "--background-image": `url(${imageURL})` }}
+          style={{ backgroundImage: `url(${imageURL})` }}
```

Comment:
What was the rationale behind this change? It's unlikely that this is the change you want to make, since we had specifically created and [use this CSS custom property](https://github.com/ShelterTechSF/sheltertech.org/blob/780ef84c827225633c8a1dc975faf4dc4f7eeae3/src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.module.css#L46) in the associated .scss file.

You can see that our SCSS uses it as part of a full [`background:` shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/background), so if you stop setting it, then that CSS property may not function correctly.

Reply:
I added as React.CSSproperties instead.